python/tk_multi_publish2/dialog.py

```diff
@@ -703,18 +732,36 @@ def _full_rebuild(self):
         Full rebuild of the plugin state. Everything is recollected.
         """
         self._progress_handler.set_phase(self._progress_handler.PHASE_LOAD)
-        self._progress_handler.push("Collecting information")
+        self._progress_handler.push(
+            "Collecting items to %s..." %
+            (self._display_action_name)
+        )
+
+        previously_collected_files = self._publish_manager.collected_files
+        self._publish_manager.tree.clear()
 
-        num_items_created = self._plugin_manager.run_collectors()
```

Comment:
new manager doesn't expose a single method to collect session and files so they're called independently here.